### DISCUSSION TITLE
Please add Korean. ㅠㅠ

### DISCUSSION THREAD

**Discussion by JEWANI** [receyuki/stable-diffusion-prompt-reader#64] [Ideas]
I'm using the program really well.
However, it's a shame that the menu and explanation are all in English.
Can't you add another language (especially Korean)?

**Comment by receyuki** [comment]
Thank you for your feedback. I've seen many articles introducing this app in various languages (including Korean), so I have considered localising the app. However,I thought the interaction of the app was simple enough, and there was very little text, so localisation seemed unnecessary. For the menu, I have provided detailed explanations in the readme (which can easily be translated using Google), what are your thoughts on this? Since you are the first user to suggest localisation, I might need more feedback before making a decision.
